processor/src/trace/virtual_bus/multilinear/mod.rs

```diff
@@ -94,6 +95,16 @@ impl<E: FieldElement> MultiLinearPoly<E> {
         *self = Self::from_evaluations(result)
             .expect("should not fail given that it is a multi-linear");
     }
+
+    pub fn project_lower_variable(&self) -> (Self, Self) {
```

Comment:
How about `project_least_significant_variable()`? Would point to the idea that the "lower" or "leftmost" variable is the least significant variable in a bitwise decomposition